src/components/Careers/InterviewProcess/index.tsx

```diff
@@ -88,7 +88,7 @@ export const InterviewProcess = () => {
                     Image={Superday}
                     title="4. PostHog SuperDay"
                     subtitle="Paid day of work"
-                    description="You’ll join a standup, meet the team, and work on a task related to your role, offering a realistic view of what it’s like working at PostHog."
+                    description="You’ll meet a few more members of the team and work on a independent project. It's challenging, but most people say it's fun!"
```

Comment:
lol "most people"

Reply:
Ha I thought about that too, do you have any ideas for saying it differently?

Reply:
"It's a test of whether you can you ship fast and still have fun"

??

feels like challenging prompts some people to unnecessarily select themselves out of the process

Reply:
```suggestion
                    description="You’ll meet a few more members of the team and work on a independent project that lets you showcase your skills."
```